packages/api/scripts/cli.js

```diff
@@ -17,12 +17,12 @@ const prog = sade('api')
 prog
   .command('build')
   .describe('Build the worker.')
-  .option('--env', 'Environment', 'dev')
+  .option('--env', 'Environment', process.env.ENV)
```

Comment:
process.env.ENV should be set, and the default here was masking a problem that the value was not being set in dev nor in CI at deploy. I will fix this more gracefully in a subsequent PR